electrum/gui/qt/util.py

```diff
@@ -407,6 +408,10 @@ def __init__(self, parent, title=None):
         if title:
             self.setWindowTitle(title)
 
+        if hasattr(parent, 'config') and isinstance(parent.config, SimpleConfig):
+            if parent.config.GUI_QT_SCREENSHOT_PROTECTION:
+                set_windows_os_screenshot_protection_drm_flag(self)
```

Comment:
This seems fragile but ok. If we wanted, we could also check if `self` has a `config`.